src/components/Editable.tsx

```diff
@@ -496,9 +497,14 @@ const Editable = ({
       // Update editingValueUntrimmedStore with the current value
       editingValueUntrimmedStore.update(value)
 
-      const { dragHold, dragInProgress } = store.getState()
+      const { dragHold, dragInProgress, showCommandMenu } = store.getState()
       if (!dragHold && !dragInProgress) {
         setCursorOnThought({ editing: true })
+
+        // Close command menu when entering edit mode on a touch device
+        if (showCommandMenu && isTouch) {
+          dispatch(toggleCommandMenuActionCreator({ value: false }))
+        }
```

Comment:
This will be covered if we move this logic to the `setCursor` reducer.

See other comment.